COLLABORATOR_GUIDE.md

```diff
@@ -334,7 +334,7 @@ documentation must state the deprecation status.
   * Emits a warning at runtime on first use of the deprecated API.
   * If used with the [`--throw-deprecation`][] flag, will throw a runtime error.
 
-* End-of-life
+* End-of-Life
```

Comment:
I thought we'd added a lint rule for this, but we forbade `End-Of-Life`. A quick search shows `End-of-life` also appears in BUILDING.md, so https://github.com/nodejs/remark-preset-lint-node/pull/13.